hack/util.sh

```diff
@@ -458,10 +458,14 @@ function install_registry() {
 }
 readonly -f install_registry
 
-function wait_for_registry() {
+function is_registry_ready() {
```

Comment:
This probably won't be used on its own apart from `wait_for_registry` function. Maybe we could make it "private" by adding `_` or `__` prefix.